packages/webapp/src/components/DateRangeSelector/index.jsx

```diff
@@ -27,10 +27,9 @@ import styles from './styles.module.scss';
 export default function DateRangeSelector({
```

Comment:
Small thing I didn't notice before -- there is a slight but handled naming conflict with dateRangeSelector from the state selectors in Finances.   `/components/Finances/DateRangeSelector/index.jsx` is at least one file that imports both {dateRangeSelector} and DateRangeSelector.

I don't have any great name alternatives haha

Reply:
Is it known to cause any problems??

Reply:
It's just a best practice to avoid having the same name in one file. The main problem with non unique names is developer confusion (in javascript). As you know this will work. Communication becomes harder.

Reply:
Understood.